tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs

```diff
@@ -78,7 +91,7 @@ private async Task BrowseButtonClick(object sender)
         }
     }
 
-    private void MoveDirectory(string sourceDirectory, string targetDirectory)
+    private int MoveDirectory(string sourceDirectory, string targetDirectory)

```

Comment:
Please don't use the C++ return HResult pattern.  Replace `MoveDirectory` with `Directory.Move`

Also, `MoveDirectory` handled the error.  The error code shouldn't be returned.

Reply:
Directory.Move cannot handle moving between drives. 

I am using the error code in the caller. What is the alternative here other than using HResult pattern?

Reply:
I think the normal C# way would be to return a bool indicating success. The `== 0` below feels very out of place.